mne_bids/mne_bids.py

```diff
@@ -641,6 +641,21 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     kind = _handle_kind(raw)
 
     bids_fname = bids_basename + '_%s%s' % (kind, ext)
+
+    # check whether the info provided indicates that the data is emptyroom
+    # data
+    emptyroom = False
+    if subject_id == 'emptyroom' and task == 'noise':
+        emptyroom = True
+        # check the session date provided is consistent with the value in raw
+        meas_date = raw.info.get('meas_date', None)
+        if meas_date is not None:
+            er_date = datetime.fromtimestamp(
+                raw.info['meas_date'][0]).strftime('%Y%m%d')
```

Comment:
I think sometimes it can also be a single element, so you might get into trouble even then

Reply:
According to the docs:
```
meas_date : tuple of int

    The first element of this list is a UNIX timestamp (seconds since 1970-01-01 00:00:00) denoting the date and time at which the measurement was taken. The second element is the additional number of microseconds.
```
So if it *can* be a single element, then the docs are not correct...

Reply:
okay fair enough. I seemed to remember from memory that it could be a single element but maybe that has changed.

@sappelhoff feel free to merge if you're happy. This PR is good to go for me